.github/workflows/lint.yml

```diff
@@ -0,0 +1,30 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Lint
+
```

Comment:
Although there's a name, is it possible to add a comment to all of the `yml` files to describe the purpose of each one and what it's testing?

Reply:
+1

Reply:
Got it!